rsts/getting_started.rst

```diff
@@ -3,107 +3,351 @@
 Getting started
 ---------------
 
-.. rubric:: Estimated time to complete: 3 minutes.
+.. rubric:: Estimated time to complete: 10 minutes.
+
 
 Prerequisites
 ***************
 
-Make sure you have `docker installed <https://docs.docker.com/get-docker/>`__ and `git <https://git-scm.com/>`__ installed, then install flytekit:
+Make sure you have `docker <https://docs.docker.com/get-docker/>`__ , `git <https://git-scm.com/>`__ and ``python > 3.6`` installed.
+
+.. caution::
+
+    We have not yet tested this flow on a Windows machine.
+
+Build -> Deploy -> Iterate mechanics
+*************************************
```

Comment:
I think we'd be better served with a full sentence here, e.g.

```
This guide will quickly take you through the mechanics of building, deploying, and iterating on Flyte tasks and workflows
```

Reply:
+1